### PR TITLE
Application sign-in summary

### PR DESCRIPTION
**Pull request by lleonard-msft**
Initial check-in for these APIs.  Changes expected.

@dkershaw10 - Please review.

### CODE REVIEW

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ca141fc](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/ca141fc97e9fcc56d6479bc484f32243ba696617)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ca141fc](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/ca141fc97e9fcc56d6479bc484f32243ba696617)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[ca141fc](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/ca141fc97e9fcc56d6479bc484f32243ba696617)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :x:Error | | [Details](#user-content-api-reference%2Fbeta%2Ftoc.yml)
[api-reference/beta/api/applicationsignindetailedsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsignindetailedsummary-get.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fapplicationsignindetailedsummary-get.md)
[api-reference/beta/api/applicationsigninsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsigninsummary-get.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fapi%2Fapplicationsigninsummary-get.md)
[api-reference/beta/resources/applicationsignindetailedsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsignindetailedsummary.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fapplicationsignindetailedsummary.md)
[api-reference/beta/resources/applicationsigninsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsigninsummary.md) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Fresources%2Fapplicationsigninsummary.md)

<a name="api-reference%2Fbeta%2Ftoc.yml"></a>
### [api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml)
 - **[Error]** W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340392) - (Line: 6728, Col: 12, Idx: 340393): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;TocDocumentProcessor&#39;: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340392) - (Line: 6728, Col: 12, Idx: 340393): While parsing a block mapping, did not find expected key..
 - **[Error]** W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340392) - (Line: 6728, Col: 12, Idx: 340393): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;ExperimentalTocDocumentProcessor&#39;: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\ivvj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340392) - (Line: 6728, Col: 12, Idx: 340393): While parsing a block mapping, did not find expected key..

<a name="api-reference%2Fbeta%2Fapi%2Fapplicationsignindetailedsummary-get.md"></a>
### [api-reference/beta/api/applicationsignindetailedsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsignindetailedsummary-get.md)
 - **[Warning]** Author arvinh-msft&quot; cannot be recognized in GitHub.
 - **Line 11**: **[Warning]** Invalid file link:(~/beta/resources/applicationSignInDetailSummary.md).
 - **Line 13**: **[Warning]** Invalid file link:(~/../concepts/permissions_reference.md).

<a name="api-reference%2Fbeta%2Fapi%2Fapplicationsigninsummary-get.md"></a>
### [api-reference/beta/api/applicationsigninsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsigninsummary-get.md)
 - **[Warning]** Author arvinh-msft&quot; cannot be recognized in GitHub.
 - **Line 14**: **[Warning]** Invalid file link:(~/../concepts/permissions_reference.md).

<a name="api-reference%2Fbeta%2Fresources%2Fapplicationsignindetailedsummary.md"></a>
### [api-reference/beta/resources/applicationsignindetailedsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsignindetailedsummary.md)
 - **[Warning]** Author arvinh-msft cannot be recognized in GitHub.
 - **Line 16**: **[Warning]** Invalid file link:(~/beta/api/applicationsignindetailedsummary_get.md).
 - **Line 17**: **[Warning]** Invalid file link:(~/beta/api/applicationsignindetailedsummary_update.md).
 - **Line 18**: **[Warning]** Invalid file link:(~/beta/api/applicationsignindetailedsummary_delete.md).

<a name="api-reference%2Fbeta%2Fresources%2Fapplicationsigninsummary.md"></a>
### [api-reference/beta/resources/applicationsigninsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsigninsummary.md)
 - **[Warning]** Author arvinh-msft cannot be recognized in GitHub.
 - **Line 18**: **[Warning]** Invalid file link:(~/beta/api/applicationsigninsummary_update.md).
 - **Line 19**: **[Warning]** Invalid file link:(~/beta/api/applicationsigninsummary_delete.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902041600287959-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=PBxsMgKXJGWBmqjpugPUc6txE8a0v7XEt6gRwcKD%2FTc%3D&st=2019-02-04T16%3A00%3A00Z&se=2019-03-07T16%3A05%3A00Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[67aae2c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/67aae2ccfdf5193e171f127ad9705828de25cba1)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[67aae2c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/67aae2ccfdf5193e171f127ad9705828de25cba1)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[67aae2c](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/67aae2ccfdf5193e171f127ad9705828de25cba1)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :x:Error | | [Details](#user-content-api-reference%2Fbeta%2Ftoc.yml)

<a name="api-reference%2Fbeta%2Ftoc.yml"></a>
### [api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml)
 - **[Error]** W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC File: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340393) - (Line: 6728, Col: 12, Idx: 340394): While parsing a block mapping, did not find expected key..
 - **[Error]** W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC File: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340393) - (Line: 6728, Col: 12, Idx: 340394): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;ExperimentalTocDocumentProcessor&#39;: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC File: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340393) - (Line: 6728, Col: 12, Idx: 340394): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;TocDocumentProcessor&#39;: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC File: W:\2axf-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6728, Col: 11, Idx: 340393) - (Line: 6728, Col: 12, Idx: 340394): While parsing a block mapping, did not find expected key..

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902041742573162-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Pr4HTlFsb4dKJnZTzLg7Ks8ZQ3%2F4TRQCr%2F%2F7ZCrVlVY%3D&st=2019-02-04T17%3A42%3A22Z&se=2019-03-07T17%3A47%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b17bb03](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/b17bb03dfc81078ee6a081aac23c3bc59e3227bc)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b17bb03](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/b17bb03dfc81078ee6a081aac23c3bc59e3227bc)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[b17bb03](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/b17bb03dfc81078ee6a081aac23c3bc59e3227bc)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :x:Error | | [Details](#user-content-api-reference%2Fbeta%2Ftoc.yml)

<a name="api-reference%2Fbeta%2Ftoc.yml"></a>
### [api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml)
 - **[Error]** W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6741, Col: 11, Idx: 340942) - (Line: 6741, Col: 12, Idx: 340943): While parsing a block mapping, did not find expected key..
 - **[Error]** W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6741, Col: 11, Idx: 340942) - (Line: 6741, Col: 12, Idx: 340943): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;TocDocumentProcessor&#39;: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6741, Col: 11, Idx: 340942) - (Line: 6741, Col: 12, Idx: 340943): While parsing a block mapping, did not find expected key..
 - **[Error]** Unable to load file &#39;beta/toc.yml&#39; via processor &#39;ExperimentalTocDocumentProcessor&#39;: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC File: W:\xnbj-s\api-reference\beta\toc.yml is not a valid TOC file, detail: (Line: 6741, Col: 11, Idx: 340942) - (Line: 6741, Col: 12, Idx: 340943): While parsing a block mapping, did not find expected key..

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902041749568102-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=qvwvvo4viKJwbjmGS3kROa768i59IFiy7wo%2BJE20yo0%3D&st=2019-02-04T17%3A50%3A20Z&se=2019-03-07T17%3A55%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6802dd7](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/6802dd7f6bd0b86016f50319702589955a28f9c8)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6802dd7](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/6802dd7f6bd0b86016f50319702589955a28f9c8)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[6802dd7](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/6802dd7f6bd0b86016f50319702589955a28f9c8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :warning:Warning | | [Details](#user-content-api-reference%2Fbeta%2Ftoc.yml)
[api-reference/beta/api/applicationsignindetailedsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsignindetailedsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/applicationsigninsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsigninsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/administrativeunit.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/administrativeunit.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsignindetailedsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsignindetailedsummary.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsigninsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsigninsummary.md) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/concepts/changelog.md) | :white_check_mark:Succeeded | |

<a name="api-reference%2Fbeta%2Ftoc.yml"></a>
### [api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml)
 - **Line 0**: **[Warning]** Invalid file link:(~/beta/api/applicationsignindetailsummary-get.md).
 - **Line 0**: **[Warning]** Invalid file link:(~/beta/resources/applicationsignindetailsummary.md).

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902041801473078-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=J847%2F%2Fl5N0A0AQggrWy3P8WyafszY1b%2Bf6sqWPRQTZQ%3D&st=2019-02-04T18%3A01%3A49Z&se=2019-03-07T18%3A06%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f63450e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/f63450e7d4d6e7e116caa5eb8f41aab6d1f6ed6c)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 23 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f63450e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/f63450e7d4d6e7e116caa5eb8f41aab6d1f6ed6c)_: 

:clock10: **Incremental building**: average incremental building time is _1 min(s) 57 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[f63450e](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/f63450e7d4d6e7e116caa5eb8f41aab6d1f6ed6c)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/applicationsignindetailedsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsignindetailedsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/applicationsigninsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsigninsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/administrativeunit.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/administrativeunit.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsignindetailedsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsignindetailedsummary.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsigninsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsigninsummary.md) | :white_check_mark:Succeeded | |
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C4%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201902041808583292-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=x5uUTw7LGJNo9uvkbb7PTdTCau5ZLbjtGT1obAcX4QA%3D&st=2019-02-04T18%3A09%3A32Z&se=2019-03-07T18%3A14%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/applicationsignindetailedsummary-get.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Retrieve the properties and relationships of [applicationSignInDetailSummary](../resources/applicationsignindetailedsummary.md) object.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/concepts/permissions-reference.md).
+
+|Permission type                        | Permissions (from least to most privileged)              |
+|:--------------------------------------|:---------------------------------------------------------|
+|Delegated (work or school account)     | AuditLog.Read.All |
```
Comment (dkershaw10) [inline api-reference/beta/api/applicationsignindetailedsummary-get.md]: This seems odd to me.  If it was audit logs, then I would expect this data to be under the auditLogs singleton.  However it's a report, under the `reports` singleton, so I would expect this to be secured using Reports.Read.All

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8418a8b](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/8418a8b1af25c27889e1cb1b3b93e77980ff3f1e)_: 

:clock10: **Preparing**: average preparing time is _1 min(s) 26 sec(s)_

File: api-reference/beta/api/applicationsignindetailedsummary-get.md (hunk 1 above)
Comment (lleonard-msft) [reply api-reference/beta/api/applicationsignindetailedsummary-get.md]: Received confirmation off-line that the permissions are correct as specified.

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8418a8b](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/8418a8b1af25c27889e1cb1b3b93e77980ff3f1e)_: 

:clock10: **Incremental building**: average incremental building time is _2 min(s) 4 sec(s)_

**Comment by jthake-msft** [comment]
OPS Build status updates of commit _[8418a8b](https://github.com/lleonard-msft/microsoft-graph-docs/pull/3916/commits/8418a8b1af25c27889e1cb1b3b93e77980ff3f1e)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/api/applicationsignindetailedsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsignindetailedsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/api/applicationsigninsummary-get.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/api/applicationsigninsummary-get.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/administrativeunit.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/administrativeunit.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsignindetailedsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsignindetailedsummary.md) | :white_check_mark:Succeeded | |
[api-reference/beta/resources/applicationsigninsummary.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/resources/applicationsigninsummary.md) | :white_check_mark:Succeeded | |
[api-reference/beta/toc.yml](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/api-reference/beta/toc.yml) | :white_check_mark:Succeeded | |
[concepts/changelog.md](https://github.com/lleonard-msft/microsoft-graph-docs/blob/update/aad-signin-summary-beta/concepts/changelog.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C201903071513322431-3916%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=dPaFgCoZn3g0K4ATZL2ij2054avXn737oBl0lggm0Es%3D&st=2019-03-07T15%3A14%3A05Z&se=2019-04-07T15%3A19%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: api-reference/beta/api/applicationsignindetailedsummary-get.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+## Example
+
+### Request
+The following is an example of the request.
+<!-- {
+  "blockType": "request",
+  "name": "get_applicationsignindetailedsummary"
+}-->
+```http
+GET https://graph.microsoft.com/v1.0/reports/applicationSignInDetailedSummary/<id>
```
Comment (dkershaw10) [inline api-reference/beta/api/applicationsignindetailedsummary-get.md]: This should say /beta

File: api-reference/beta/api/applicationsigninsummary-get.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | AuditLog.Read.All |
+|Delegated (personal Microsoft account) | Not supported   |
+|Application | AuditLog.Read.All | 
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+``` http
+
```
Comment (dkershaw10) [inline api-reference/beta/api/applicationsigninsummary-get.md]: You are missing the actual request here!

File: api-reference/beta/api/applicationsignindetailedsummary-get.md (hunk 2 above)
Comment (lleonard-msft) [reply api-reference/beta/api/applicationsignindetailedsummary-get.md]: Fixed.

**Comment by lleonard-msft** [comment]
Due to unresolvable merge conflicts, this PR will be closed.  

PR #4193 is the new PR for this content.

**Pull request closed without merging**